scripts/docker_deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+if [ "$TRAVIS_BRANCH" == "develop" ]; then
+	version=`grep -oP "PROJECT_VERSION \"?\K[0-9.]+(?=\")"? CMakeLists.txt`;
+	docker tag ethereum/solc:build ethereum/solc:nightly;
+	docker tag ethereum/solc:build ethereum/solc:"$version"-"$TRAVIS_COMMIT"
+	docker push ethereum/solc:"$version"-"$TRAVIS_COMMIT";
+	docker push ethereum/solc:nightly;
+elif [ "$TRAVIS_PULL_REQUEST_BRANCH" == "master"]; then
```

Comment:
If we are currently building a pull request, we should never publish anything anywhere.